src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_DebugInfoScreen.tsx

```diff
@@ -572,6 +593,22 @@ export const Profile_DebugInfoScreen = () => {
             }
           />
         </Section>
+
+        {isBeaconsEnabled && KETTLE_API_KEY && (
```

Comment:
Should be `!!KETTLE_API_KEY` to ensure a boolean value I think

Reply:
KETTLE_API_KEY is a string but might be undefined, this condition just to make sure is not undefined.

Reply:
Fixed!!

Reply:
Nice!!